### PR TITLE
Use tex-svg-full.js for MathJax3

### PR DESCRIPTION
**Pull request by j-fu**
This provides access to mhchem and the chemistry arrows used e.g. by Catalyst.jl.
@pjaap, @DA-BE-RU

### CODE REVIEW

**Comment by j-fu** [comment]
Also, this would bethe same version as used by [Pluto.jl](https://github.com/fonsp/Pluto.jl/blob/34409a7c5468b2ae1f0c8f26af099cbf5293220a/frontend/editor.html#L45)
@rikhuijzer

**Comment by mortenpi** [comment]
Do we know how much heavier this makes the MathJax dependency?

**Comment by j-fu** [comment]
```
ls -l tex-svg*.js
-rw-rw-r-- 1 fuhrmann users 2275113 2024-08-07 09:34 tex-svg-full.js
-rw-rw-r-- 1 fuhrmann users 2108580 2024-08-07 09:33 tex-svg.js
```

These are just the file sizes. So IMHO not a huge difference. Not sure what the additional setup with `mhchem` would involve.

**Comment by mortenpi** [comment]
That seems fine to me, so LGTM! Do you mind adding a note about this into the CHANGELOG.md as well?

**Review by rzyu45 — APPROVED** [review]

**Comment by j-fu** [comment]
bump :)

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,18 @@
...
 
+## Version [v1.X.Y] - YYYY-MM-DD
+
+### Added
+
+### Changed
+
+* The MathJax3 setup now uses `tex-svg-full.js` and additionally draws in `mhchem` by default ([#2549]). 
+  This new default allows to render chemical arrows as generated by Catalyst.jl without additional 
+  setup. It is also consistent with the setup used by Pluto.jl.
```
Comment (mortenpi) [inline CHANGELOG.md:14]:
```suggestion
* The MathJax3 setup now uses `tex-svg-full.js` and additionally draws in `mhchem` by default, allowing for chemistry symbols to be rendered (consistent with Pluto.jl). ([#2549])
```

**Comment by mortenpi** [comment]
Sorry for the delay. Let's get this merged and tagged!

**Pull request merged**